static/app/components/events/interfaces/debugMeta-v2/debugImageDetails/candidate/actions.tsx

```diff
@@ -147,15 +147,15 @@ export default Actions;
 const StyledDropdownLink = styled(DropdownLink)`
   display: none;
 
-  @media (min-width: ${props => props.theme.breakpoints[2]}) {
+  @media (min-width: ${props => props.theme.breakpoints[4]}) {
     display: flex;
     align-items: center;
     transition: none;
   }
 `;
 
 const StyledButtonBar = styled(ButtonBar)`
-  @media (min-width: ${props => props.theme.breakpoints[2]}) {
+  @media (min-width: ${props => props.theme.breakpoints[4]}) {
```

Comment:
Breakpoint 4 is 2560px, is that too soon to start hiding some of the UI?

Reply:
the only major difference is the buttons. on smaller devices, the UI displays two buttons and on larger devices the ellipsis button, so I think it's fine